api/users.js

```diff
@@ -190,3 +196,40 @@ router.patch("/me/password", authenticate, async (req, res, next) => {
     next(e);
   }
 });
+
+//GET logged in user interests for suggestion component
+router.get("/me/interests", authenticate, async (req, res, next) => {
+  try {
+    const myInterests = prisma.user.findUniqueOrThrow({
+      where: { id: req.user.id },
+      include: {
+        interests: true,
+      },
+    });
+    res.json(myInterests);
```

Comment:
At the moment, this endpoint is returning the whole user object. What if we just returned the interests part? Otherwise, there's no difference between this endpoint and the "/me" endpoint

For example:
`const user = prisma.user.findUniqueOrThrow...`
`res.json(user.interests)`

Same would apply to the "/:id/interests" endpoint

Reply:
got it, yeah was thinking about that when going over the frontend code